administrator/admin.py

```diff
@@ -3,4 +3,26 @@
 
 from django.contrib import admin
 
-# Register your models here.
+from restaurant.models import Restaurant
+
+
+def soft_delete(modeladmin, request, queryset):
+    """Soft delete function for selected restaurants in QuerySet list"""
```

Comment:
Please, fix this according to PEP 257.

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".
[PEP 257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)